main/f2fs-tools/APKBUILD

```diff
@@ -36,34 +36,23 @@ snapshot() {
 	scp $pkgname-$pkgver.$_format $disturl || return 1
 }
 
-prepare() {
-	local i
-	cd "$_builddir"
-	for i in $source; do
-		case $i in
-		*.patch) msg $i; patch -p1 -i "$srcdir"/$i || return 1;;
-		esac
-	done
-	libtoolize --force && aclocal && autoheader && autoconf \
-		&& automake --add-missing
-}
-
 build() {
-	cd "$_builddir"
+	cd "$builddir"
+	
+	./autogen.sh
```

Comment:
Could you move the `autogen.sh` invocation to the prepare stage by adding a `prepare()` function the APKBUILD? Also make sure to call default_prepare and make sure you change the directory afterwards.

Reply:
Sure.